packages/cactus-plugin-ledger-connector-besu/package.json

```diff
@@ -86,6 +86,7 @@
     "@hyperledger/cactus-common": "0.5.0",
     "@hyperledger/cactus-core": "0.5.0",
     "@hyperledger/cactus-core-api": "0.5.0",
+    "@hyperledger/cactus-plugin-consortium-manual": "0.5.0",
```

Comment:
@AzaharaC We cannot depend on a specific implementation of the consortium plugin. Please use the `IPluginConsortium` interface instead when looking up the consortium plugin instance in the plugin registry and remove the manual consortium plugin from the list of dependencies.

Reply:
To do this, I have changed the code for this:
```
if (nodeHostsProvider === NodeHostsProviderType.ConsortiumPlugin) {
      const consortium = this.pluginRegistry.plugins.find(
        (plugin) => plugin.getInstanceId() == consortiumPluginId,
      ) as IPluginConsortium<unknown, unknown, unknown, unknown>;
```
Then I have this line: ```const nodesAll = await consortium.getConsortiumJws();``` When I want to use the constant ```nodesAll ``` it gives me an error saying that it is unknown type and I can't access its properties.

To use this variable I think I would have to do something like this: ```const nodesAll: GetConsortiumJwsOut = await consortium.getConsortiumJws();```, but I would need to import ```GetConsortiumJwsOut``` from the consortium plugin and we would return to the initial dependency problem

Reply:
@AzaharaC You can cast it to `any` initially until a better solution is available (you'll lose auto-completion/type info but it will compile)